src/components/DeleteAccount/DeleteAccount.jsx

```diff
@@ -42,11 +43,23 @@ const DeleteAccount = () => {
   const onError = error => {
     setStatus(IDLE)
     console.error(error.message) // eslint-disable-line no-console
-    Alerter.error(t('DeleteAccount.error.message'))
+    showAlert({
+      message: t('DeleteAccount.error.message'),
+      type: 'error'
+    })
   }
 
-  const onRequested = () => {
-    setStatus(EMAIL_CONFIRMATION)
+  const onRequested = ({ byEmailOnly }) => {
+    if (byEmailOnly) {
+      showAlert({
+        message:
+          'La demande a été envoyée et sera traitée rapidement par nos équipes.',
```

Comment:
this should be translated i guess.

edit: i found the string should be byEmailOnly.success